scripts/start_integration_test_contracts_with_docker.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Get the latest Finschia tag
+TAG=$(curl "https://api.github.com/repos/Finschia/finschia/tags" | jq -r '.[0].name')
+
+# Copy init_single.sh from Finschia/finschia/init_single.sh
+init_single=$(mktemp)
+curl "https://raw.githubusercontent.com/Finschia/finschia/${TAG}/init_single.sh" -o $init_single
+
+# Remove v from tag prefix
+# ex. v1.0.0 -> 1.0.0
+TAG=$(echo "$TAG" | cut -c 2-)
+TEST_DOCKER_IMAGE=finschia/finschianode:${TAG}
+docker pull finschia/finschianode:${TAG}
+
+export FNSAD="docker run -i --rm -p 26656:26656 -p 26657:26657 -v ${HOME}/.finschia:/root/.finschia ${TEST_DOCKER_IMAGE} fnsad"
+
+# run Finschia/finschia/init_single.sh
+bash -xe $init_single
```

Comment:
Should it execute with `-xe`? In other words, does this verbose output help users using this script for tests?

Reply:
Thanks. This verbose is not so helpful for users. I removed it.